assets/xml/misc/z_select_static.xml

```diff
@@ -0,0 +1,42 @@
+<Root>
+    <!-- OOTSW97 is currently being used here because ZAPD can't manually select f3dex. A fix for this is in a ZAPD PR. When it is merged into OOT this updated XML can be used here
+    https://gist.github.com/louist103/75d7a58f9890652d02feb6702bc4b2e0-->
```

Comment:
should we just wait for ZAPD to merge the fix and be updated in oot then, since this PR has no other content

Reply:
zap has been updated in a recent pr so this is probably fixable now? didnt keep up if the fix was in that round of changes though

Reply:
ZAPD has been updated but its not in master yet.  I would like to see a few more PRs for it get in before bringing it in since this PR isn't too important.

Reply:
elliptics mac pr brought in updates, which is what im thinking of. that isnt merged yet but it will be soon

Reply:
Looks like his PR has that.  When it gets merged ill update this.